TODO.rst

```diff
@@ -0,0 +1,26 @@
+Planned features
+================
+
+Skeleton TODO
+-------------
+
+ - Plugin class definitions
+ - Hooks?
+ - Logging in general
+ - Logging in tests
+ - sketch out kolibri command
+ - implement settings pattern
+ - python setup.py sdist bdist_wininst upload
+ - versioning
+ - changelog
+ - Circle repo
+
+Decision TODO
+-------------
+
+#. Django 1.8 or 1.9 (LTS or latest?)
+#. CircleCI, right?
+#. Should JS / BDD tests be separated out? How to run optionally?
```

Comment:
I'm thinking about just adding `tox -e bdd` as an example and try to see if I can get a conditional test script implemented. This way, we won't have to install selenium for the other tests, because Tox can manage to only install it when running the `bdd` environment. Same thing for the current `tox -e lint`.